docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.1'
+services:
+  npm:
+    image: 'node:latest'
```

Comment:
why even use docker-compose?
also i'm kinda wary of using 'latest' tags, usually better to have a version pinned (maybe also using .nvmrc?)

Reply:
we can keep it if you're using it that way - no harm in it
i'll probably add a `.nvmrc` for me though as we have a version pinned now :)